scripts/publish/aws-s3.sh

```diff
@@ -31,20 +31,23 @@ function usage() {
   echo "    -b <s3 bucket>"
   echo "    -v <version>"
   echo "    -p <product name>"
+  echo "    -k <S3 key prefix>"
```

Comment:
I suggest to keep it as `-k [s3 key prefix]` for now, as there are other scripts using similar notation that we would have to update as well, and fall beyond the scope of this PR. I want to split the scripts to a separate repository sometime soon (since the resin desktop project needs them), so we can do a full cleanup there.